client/store/reports/stats/index.js

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import actions from './actions';
+import selectors from './selectors';
+import resolvers from './resolvers';
+
+export default {
+	actions,
+	selectors,
+	resolvers,
```

Comment:
reducers isn't imported/exported here, so I'm getting an error in the browser `No reducer provided for key "stats"`

Reply:
good catch! 🤦‍♂️